pkg/jsonapi/api_test.go

```diff
@@ -174,31 +95,61 @@ sub:
   subentry: 123
 `)},
 	}
-
-	totp, _, err := otp.Calculate(context.Background(), "_", totpSecret)
-	if err != nil {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// this fails if the time second switches between `otp.Calculate` below and in pkg/jsonapi/responses
+	// if this is the case a different (wrong) totp is generated
+	// we need this in order to make it testable: https://github.com/gokyle/twofactor/pull/17
+	ok := gptest.Retry(t, 10, time.Millisecond*5, func(r *gptest.R) {
```

Comment:
This is a rare edge case that happens on about 1% of tests

Reply:
This is due to Time based OTP tokens are time based ;-) - it's ok here for now unless we mock out time dependency.